tests/edge-cases/not-05.txt

```diff
@@ -0,0 +1,2 @@
+brun '(not (q (1 2)))'
```

Comment:
`brun '(not (q (1 2)))'`
@richardkiss This seems like it should be an error to me.

Reply:
This is a boolean operator, so it ought to have the same semantics as `i`. This is how `i` works too.